fact/time.py

```diff
@@ -30,112 +20,126 @@
 OFFSET = (datetime(1970, 1, 1) - datetime(1, 1, 1)).days
 
 UNIX_EPOCH = datetime(1970, 1, 1, 0, 0, tzinfo=timezone.utc)
-MJD_EPOCH = datetime(1858, 11, 17, 0, tzinfo=timezone.utc)
-MJD_EPOCH_NUMPY = np.array(MJD_EPOCH.timestamp()).astype('datetime64[s]')
+MJD_EPOCH = datetime(1858, 11, 17, 0, 0, tzinfo=timezone.utc)
+
+
+def datetime_to_numpy(dt):
+    '''
+    Convert a python datetime object to numpy.datetime64
+    '''
+    return np.array(dt.timestamp()).astype('datetime64[s]')
 
 
 def unixtime_to_mjd(unixtime):
+    '''
+    Convert a unix timestamp to mjd
+    '''
     return (unixtime - (MJD_EPOCH - UNIX_EPOCH).total_seconds()) / 3600 / 24
 
 
 def mjd_to_unixtime(mjd):
+    '''
+    Convert an mjd timestamp to unix
+    '''
     return (mjd + (MJD_EPOCH - UNIX_EPOCH).total_seconds()) * 3600 * 24
 
 
-def datetime_to_mjd(dt):
+def datetime_to_mjd(dt, epoch=MJD_EPOCH):
+    '''
+    Convert a datetime to julian date float.
+    This function can handle python dates, numpy arrays and pandas Series.
+
+    Parameters
+    ----------
+    dt: datetime, np.ndarray[datetime64], pd.DateTimeIndex, pd.Series[datetime64]
+        The datetime object to convert to mjd
+    epoch: datetime
+        The epoch, default is classic MJD (1858-11-17T00:00)
```

Comment:
under what circumstances would users put something else than `epoch=MJD_EPOCH` here given that the function is called `datetime_to_mjd` ? 

Could you give a little example in the doc-string?

Reply:
See below, FACT uses the unix epoch as base for our `MJD`.

Reply:
Thx